freetextresponse/freetextresponse.py

```diff
@@ -178,10 +178,8 @@ def workbench_scenarios():
             'This is the message students will see upon '
             'submitting a draft response'
         ),
-        default=_(
-            'Your answers have been saved but not graded. '
-            'Click "Submit" to grade them.'
-        ),
+        default='Your answers have been saved but not graded. '
```

Comment:
Let's at least still wrap this in parens;
the indenting is wonky with the implicit line continuation.